src/platforms/web/util/model.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+
+export function getModelModifier (vnode: VNodeWithData): ASTModifiers | void {
+  const directives = vnode.data.directives || []
+  for (let i = 0, directive; i < directives.length; i++) {
+    directive = directives[i]
+    if (directive.name === 'model') {
+      return directive.modifiers
+    }
+  }
+}
+
+export function isToNumber (modifiers: ?ASTModifiers, type: ?string): boolean {
```

Comment:
I guess you can use a better name, as `isToNumber` isn't the most meaningful of them IMHO.